modules/041-linstor/templates/linstor-affinity-controller/deployment.yaml

```diff
@@ -95,6 +95,13 @@ spec:
                 secretKeyRef:
                   name: linstor-client-https-cert
                   key: ca.crt
+          resources:
+            requests:
+              {{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 14 }}
+              {{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+              {{- include "linstor_affinity_controller_resources" . | nindent 14 }}
```

Comment:
```suggestion
                {{- include "linstor_affinity_controller_resources" . | nindent 14 }}
```

Reply:
internally decided to keep `nindent` always in correct postion